package.json

```diff
@@ -53,15 +53,14 @@
     "grunt-coveralls": "^1.0.1",
     "grunt-fh-build": "2.0.0",
     "istanbul": "^0.4.3",
-    "mocha": "^2.2.5",
+    "mocha": "^5.1.1",
```

Comment:
Is there any impact from bumping mocha here?

Reply:
The mocha is used in the tests CI, and the upper version of mocha is required to work with NodeJS 8.